docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -61,6 +61,18 @@ if set too high.
 
     Only applies for ``uniform`` :ref:`scheduler policy <node-scheduler-policy>`.
 
+``node-scheduler.max-unacknowledged-splits-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``2000``
+
+Maximum number of splits that are either queued on the coordinator, but not yet sent or confirmed to have been received by
+the worker, or assigned to the tasks in the current scheduling batch, but not yet added to the coordinator-local RemoteTask. This limit
```

Comment:
`or assigned to the tasks in the current scheduling batch, but not yet added to the coordinator-local RemoteTask` is not needed